### PR TITLE
refactor(site): Refactor template settings

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Before:
Screen Shot 2023-02-16 at 13 56 05

After:
Screen Shot 2023-02-16 at 13 56 13
Screen Shot 2023-02-16 at 13 56 25

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]
@bpmct could to review the UX + copy for this too?

**Comment by BrunoQuaresma** [comment]
The copy used is the same as the "Create template" page but feel free to suggest anything @bpmct

File: site/src/i18n/en/templateForm.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "generalInfo": {
+    "title": "General info",
+    "description": "The name is used to identify the template in URLs and the API. It must be unique within your organization."
```
Comment (bpmct) [inline site/src/i18n/en/templateForm.json:4]:
```suggestion
    "description": "The name is used to identify the template in URLs and the API. It must be unique."
```

File: site/src/i18n/en/templateForm.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "title": "Schedule",
+    "description": "Define when workspaces created from this template automatically stop."
+  },
+  "operations": {
+    "title": "Operations",
+    "description": "Regulate actions allowed on workspaces created from this template."
+  },
+  "parameters": {
+    "title": "Template params",
+    "description": "These params are provided by your template's Terraform configuration."
```
Comment (bpmct) [inline site/src/i18n/en/templateForm.json:20]:
```suggestion
    "title": "Template variables",
    "description": "These variables are provided by your template's Terraform configuration."
```

**Review by bpmct — APPROVED** [review]
Copy LGTM

**Review by Kira-Pilot — APPROVED** [review]

**Pull request merged**